acceptance/tests/i18n/translation_fallback.rb

```diff
@@ -0,0 +1,19 @@
+test_name 'C100560: puppet agent run output falls back to english when language not available' do
+  # No confines because even on non-translation supported OS' we should still fall back to english
+
+  tag 'audit:medium',
+      'audit:acceptance'
+
+  agents.each do |agent|
+    step 'Run Puppet apply with language French and check the output' do
+      unsupported_language='fr_FR'
```

Comment:
Should we maybe pick something that is not a real locale here? I could see a world in which this test starts to fail because we add French translations.

Reply:
I thought about that, and I can go either way. If we choose an unsupported string, then we short circuit because its not a supported locale. If we leave it along, then in some cases the language is supported, but we don't have a translation for it.

Reply:
Ahh I see. Alright. Then maybe pick a language that's less likely to be high on the translation list? (There are already some real French translations in Leatherman) We could show a little Hungarian love with "hu_HU" :)